download.sh

```diff
@@ -12,8 +12,7 @@ GEOSERVER_VERSION=$1
 TEMP_DOWNLOADED=$2 
 
 echo "GeoServer Data Directory is going to be downloaded"
-# for debugging
-echo "curl -k -L -O https://www.dropbox.com/s/q0qc2t7d9alo9fk/data-${GEOSERVER_VERSION}.zip"
-curl -k -L -O https://www.dropbox.com/s/q0qc2t7d9alo9fk/data-$GEOSERVER_VERSION.zip && \
-unzip -x -d ${TEMP_DOWNLOADED} data-$GEOSERVER_VERSION.zip
+artifact_url="https://www.dropbox.com/s/q0qc2t7d9alo9fk/data-$GEOSERVER_VERSION.zip"
```

Comment:
@randomorder I suspect the dropbox link must be updated here too

Reply:
Yes, PR updated
thanks!